src/foam/nanos/cron/Schedulable.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.cron',
+  name: 'Schedulable',
```

Comment:
This model should extend Cron and implement ContextAgent, as it will be run as a cron job.  
It's runScript will simple submit itself to the threadpool agency which will call 'execute' which will do the work of submitting the entry to the dao. 
And additionally, this gives us a model from which to extend and add additional logic in 'execute' should the put require extra setup or exception handling.

Reply:
Done. Left as Todo's for @nanoNeel to implement the runScript and execute method. But enough to unblock ruby.